### ISSUE TITLE
Key Binds Profiles

### ISSUE THREAD

**Issue by FoUnDeRR** [multitheftauto/mtasa-blue#3497]
### Is your feature request related to a problem? Please describe.

I'm always frustrated when trying to change key bind settings from one server to another. There are dozens of online servers exists with different game mechanics, most people attempt to have custom key binds to each server. And when connecting to another server, it'd be a problem since many key binds are custom-set and would push errors in other servers.

### Describe the solution you'd like

Create a button before the Load Defaults button in Binds settings, called "Profiles", "Custom Profiles", or any other name as you like. And make an interface where we can create profiles for the key binds. 

When creating key bind profiles, make it possible to name set names for each. Then also create a button called Load Profile and we can select any profile we want and load it. 

For example, in my default profile, Fire key is set to M key, but M key is locked in another server. So, I can create another profile where I can set it to F. After that, I can switch bind profiles that will fit to my servers I'm playing on.  

### Describe alternatives you've considered

Since this is not possible, 
I used to create a .txt file and have all my key binds there. When switching servers, I re-add those key binds manually to the game continuously which is really exhausting and overwhelming. That precious amount of time could be used to do something creative or just casually hangout with friends, but it is not doable since I'm busy dealing with this issue.  

### Additional context

Many MTA servers have custom controls, bound to each key in the keyboard, especially popular and massive communities like FFS Gaming.

Think like this,
I played in a server for about 5 years with many custom bound keys and mechanics just for the server. And I get bored after a while playing there, so I attempt to play on another server. After joining that server, every button I'd press would be malfunctioning since half of the keys are custom bound to that previous server.

The biggest issue would be some servers that don't accept the issue would punish players from automated moderating systems like auto-mute/auto-kick/auto-ban.

Hopefully I'll be able to see a solution or similar one for this issue, 
Sincerely yours,
Grzzly, aka FoUnDeR.

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not about a cheat or security vulnerability.

**Comment by T-MaxWiese-T** [comment]
Which key bindings are you talking about? There are various, for example, GTA controls or key bindings for resources. Fire, for example, belongs to the GTA controls. Resource button bindings in the MTA settings only work if buttons are bound to commands. This means that currently it is probably not possible to change most of the button bindings for resources. So profiles would make sense for server-defined controls.

**Comment by FoUnDeRR** [comment]
Yes, I meant GTA controls, it will work for server-defined ones too.
